tests/test_meshtastic_utils_connect_paths.py

```
@@ -357,6 +360,464 @@ def test_connect_meshtastic_logs_firmware_version_on_success(
     )
 
 
+def test_connect_meshtastic_bootstraps_skew_for_fast_receive_during_subscribe(
+    reset_meshtastic_globals,
```

Ruff (0.15.9)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)